windows/gui-source/KA Lite/KALite.cpp

```diff
@@ -209,6 +211,25 @@ void autoStartServerAction()
 	}
 }
 
+void showKaliteServerLogs()
+{	
+	const DWORD MAX_SIZE = 255;
+	std:string filePath = "\\server.log";
+	char home_path[MAX_SIZE];
+	kaliteHomePath(home_path, MAX_SIZE);
+	std::string str(home_path);
+	struct stat fileAtt;
+	std::string kalite_logs_path = home_path + filePath;

```

Comment:
Inconsistent variable naming - this is C++ and not Python - your variables should use camelCase.  Here's an MSDN reference that you can apply: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions.